### PR TITLE
convert build command to use async-await

### PR DESCRIPTION
**Pull request by FredKSchott**
no behavior change, just a big indentation change and cleaner code

/cc @justinfagnani

### CODE REVIEW

File: src/build/build.ts [hunk 1]
```diff
@@ -40,105 +40,100 @@ export interface BuildOptions {
...
-    let bundledPostProcessing = Promise.all([loadSWConfig, waitFor(bundledPhase)]).then((results) => {
-      let swConfig: SWConfig = results[0];
-      return addServiceWorker({
-        buildRoot: 'build/bundled',
-        project: polymerProject,
-        swConfig: swConfig,
-        bundled: true,
-      });
+  // Once the unbundled build stream is complete, create a service worker for the build
+  let unbundledPostProcessing = Promise.all([loadSWConfig, waitFor(unbundledPhase)]).then((results) => {
```
Comment (justinfagnani) [inline src/build/build.ts]: I don't suppose it would be any nicer to use an IIAFE?

```typescript
(async () => {
  const result = await Promise.all([loadSWConfig, waitFor(unbundledPhase)]);
  const swConfig = result[0];
  await addServiceWorker(...);
}());
```

Just brainstorming to see if there's any nice pattern for doing all this parallel work...

Comment (FredKSchott) [inline src/build/build.ts]: Normally I would agree, but since the result is being saved to `unbundledPostProcessing` as a promise and then used later inside of a `Promise.all()` call I kind of like that we're explicitly using a Promise here.

tl;dr: ¯\_(ツ)_/¯
